submissions/SkirnevskyiAleksandr/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <!--header-->
+    <header class="container">
+     <!--header-menu-->
+        <menu toolbar class="menu">
+            <li class="menu-item"><a class="menu-item" href="#">Mail</a></li>
+            <li class="menu-item"><a class="menu-item" href="#">Images</a></li>
+            <li class="menu-item">
+    <!--dropdown-btn-->            
+                <input type="checkbox" id="drop-down-btn" class="dropDownInput">
+                <label class="dropDownInput" for="drop-down-btn"><img src="./icons/popup-button.png"
+                        alt="popup-button"></label>
+    <!--dropdown-menu-->                
+                <ul class="DropDownMenu">
+
```

Comment:
Could you please remove all empty lines inside of markup? You can format a code by auto-formatting (option at your editor/IDE. (you can install into your editor as an extension if you do not have yet))